core/util/src/main/java/orca/util/db/MySqlBase.java

```diff
@@ -154,6 +155,14 @@ public static String constructQueryPartial(Properties p)
      */
     @Persistent(key = PropertyMySqlPassword)
     protected String mySqlPasswd;
+    /**
+     * MySql Connections Options. Default empty.
+     * e.g. "&verifyServerCertificate=false&useSSL=true"
+     * https://github.com/RENCI-NRIG/exogeni/issues/173
+     * https://github.com/RENCI-NRIG/orca5/pull/156
+     */
+    @Persistent(key = PropertyMySqlConnectionOptions)
+    protected String mySqlConnectionOptions;
```

Comment:
Assign an empty string to avoid it being null (there are three cases - property not specified, specified and empty and specified and has a value)